spec/models/storage_spec.rb

```diff
@@ -505,4 +505,24 @@
       expect(storage.smartstate_analysis_count_for_host_id(host.id)).to eq(2)
     end
   end
+
+  context '#storage_clusters' do
+    it 'returns only parents of type storage_cluster' do
+      # A storage mounted on different VCs will have multiple parents
+      storage          = FactoryGirl.create(:storage,               :name => 'test_storage_vmfs', :store_type => 'VMFS')
```

Comment:
This StorageCluster is a concept not limited to VMware only.  So I think keeping the test case to generic storage. What do you think?